tests/runtimes/local/models/models_test.py

```diff
@@ -609,3 +609,72 @@ def testGetAgentGroupsByAssetType_always_retrievesAgentGroupsByAssetType(
         assert agent_group_2.description in [
             group.description for group in agent_groups_link
         ]
+
+
+def testWhenNewVulnerabilityReported_always_scanRiskRatingIsUpdated(
+    mocker: plugin.MockerFixture, db_engine_path: str
+) -> None:
+    """Test updating the scan risk rating when a new vulnerability is reported."""
+    mocker.patch.object(models, "ENGINE_URL", db_engine_path)
+    scan = models.Scan.create(title="Test Scan")
+
```

Comment:
This is too long, you should just move the first db persist to a fixture.

Reply:
I think its clear like that, in first its info and later critical since another vuln reported.